jaeger_client/sampler.py

```diff
@@ -280,14 +281,19 @@ def __init__(self, strategies, max_operations):
     def is_sampled(self, trace_id, operation=''):
         sampler = self.samplers.get(operation)
         if not sampler:
-            if len(self.samplers) >= self.max_operations:
+            sampler = self.per_operation_default_samplers.get(operation)
+        if not sampler:
+            if (
+                len(self.samplers) + len(self.per_operation_default_samplers) >=
```

Comment:
note for reviewers: with the current configuration flake8 complains whether the line break is either before _or_ after this operator.  I think this means that either W503 or W504 needs to be disabled depending on the project's preference, or `ignore` should be changed to `extend-ignore`?   https://gitlab.com/pycqa/flake8/issues/466

Reply:
@yurishkuro Any thoughts on which way to go on this one?

Reply:
You can extract the longest expression into a variable.

Reply:
I guess I meant more in terms of fixing the mutually contradictory configuration of the style checking in CI.  As you say though, I can certainly work around it in this case.